### PR TITLE
Ft group chat 157801302

### PR DESCRIPTION
**Pull request by saheedt**
#### What does this PR do?
This PR ensures users can have a group chat while playing a game 
#### Description of Task to be completed?
- Users should be able enter a group chat 
#### How should this be manually tested?
- Input message in the provided chat panel on the game screen
#### Screenshots (if appropriate)
screen shot 2018-06-24 at 12 13 18 am
screen shot 2018-06-24 at 12 13 48 am
screen shot 2018-06-24 at 12 14 09 am

#### What are the relevant pivotal tracker stories?
- #157801302

### CODE REVIEW

File: app/controllers/index.js [hunk 1]
```diff
@@ -1,23 +1,22 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    _ = require('underscore');
+var mongoose = require('mongoose'), //eslint-disable-line
```
Comment (fortunee) [inline app/controllers/index.js]: I'm not sure why you have our `eslint` disabled. 
Please let's enable them and fix all lint errors. @saheedt

File: app/controllers/index.js [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
 
-exports.render = function(req, res) {
-    res.render('index', {
-        user: req.user ? JSON.stringify(req.user) : "null"
-    });
-};
\ No newline at end of file
+exports.render = function(req, res) { //eslint-disable-line
+  res.render('index', {
+    user: req.user ? JSON.stringify(req.user) : 'null'
```
Comment (fortunee) [inline app/controllers/index.js]: Why do we have `null` in quotes? I think It should be just `null` and not a string. @saheedt

File: public/css/style.css [hunk 3]
```diff
@@ -389,6 +393,17 @@
    animation-timing-function: ease-in-out;
  }
 
+ .chat-panel-slide-up {
+  /* animation-duration: 0.7s; */
```
Comment (fortunee) [inline public/css/style.css]: Remove this and every other commented out code @saheedt

**Review by fortunee — COMMENTED** [review]
@saheedt @OlatunjiYso Please review my comments

**Comment by saheedt** [comment]
@fortune Review has been carried out. All comments cleared and eslint re-activated.

**Comment by fortunee** [comment]
Except now we have merge conflicts. Please resolve them @saheedt @OlatunjiYso

**Comment by saheedt** [comment]
@fortunee conflicts resolved, we are open for review

**Pull request merged**